### DISCUSSION TITLE
开启VPN后，哔哩哔哩UWP应用就用不了，怎么解决这个呢

### DISCUSSION THREAD

**Discussion by gcl-l** [ywmoyue/biliuwp-lite#690] [Q&A] [answered]
### Bug 描述

可以不通过代理直接用哔哩哔哩UWP吗，
或者VPN软件屏蔽什么网址设置

### 操作步骤

```plain text
.
```

### 期望结果

.

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment, marked as answer]
https://github.com/ywmoyue/biliuwp-lite/discussions/616

是uwp loopback的问题, 在软件内即可找到相关教程

**Reply by gcl-l** [reply]
> #616
> 
> 是uwp loopback的问题, 在软件内即可找到相关教程

感谢，成功了，绕过UWP应用网络代理就行了
